app/views/idv/inherited_proofing/get_started.html.erb

```diff
@@ -59,4 +59,12 @@
           ),
         ) %>
   </p>
+
+  <% if user_fully_authenticated? %>
+      <%= render 'shared/cancel', link: idv_inherited_proofing_cancel_path(step: 'get_started') %>
```

Comment:
How does the user get into the flow if they are not fully authenticated? It seems like that shouldn't be possible and we should have something in the controller to prevent that

Reply:
@jmhooper  Correct. I'll take that out.

Reply:
Done.